src/job_handler_plugins/radix/__init__.py

```diff
@@ -35,7 +35,8 @@ def start(self) -> str:
         # so that we can call the job scheduler
         # to get the progress or to remove the job.
         self.job.state = {"job_name": result.json()["name"]}
-        return result.status()  # type: ignore
+        logger.info(f"result:  {result}")
```

Comment:
I guess this could be a lot of data. Perhaps set it behind "logger.debug"?
So not to pollute the logs